server/routes/makeAReferral/delete/probation-practitioner-email/deleteProbationPractitionerPresenter.ts

```diff
@@ -0,0 +1,28 @@
+import PresenterUtils from '../../../../utils/presenterUtils'
+
+export default class DeleteProbationPractitionerPresenter {
+  backLinkUrl: string
+
+  constructor(
+    private readonly id: string,
+    private readonly crn: string,
+    private readonly ndeliusPPEmailAddress: string | null | undefined,
+    private readonly firstName: string | null = null,
+    private readonly lastName: string | null = null,
+    private readonly userInputData: Record<string, string> | null = null
+  ) {
+    this.backLinkUrl = `/referrals/${id}/confirm-probation-practitioner-details`
+  }
+
+  readonly text = {
+    title: 'Are you sure you want to delete the probation practitioner email address?',
+    label: `${this.firstName} ${this.lastName} (CRN: ${this.crn})`,
+    inputHeading: 'Email Address',
+  }
+
+  private readonly utils = new PresenterUtils(this.userInputData)
+
+  readonly fields = {
+    ndeliusPPEmailAddress: this.utils.stringValue(this.ndeliusPPEmailAddress!, 'delius-probation-practitioner-email'),
```

Comment:
we dont' need to define the fields as it is not required for delete functionality